template/src/project_name/accounts/tests.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth import get_user_model
+
+from .models import User as UserModel
+
+
+def test_create_user() -> None:
+    User = get_user_model()
+    user = User.objects.create_user(username="test", password="123")  # noqa: S106
+
+    assert isinstance(User, UserModel)  # noqa: S101
+    assert user.username == "test"  # noqa: S101
+    assert user.is_active  # noqa: S101
+    assert not user.is_staff  # noqa: S101
+    assert not user.is_superuser  # noqa: S101
+    assert user.email == ""  # noqa: S101
+
+
+def test_create_superuser() -> None:
+    User = get_user_model()
+    admin_user = User.objects.create_superuser(  # noqa: S106
+        username="superuser", password="123"
+    )
+
+    assert isinstance(User, UserModel)  # noqa: S101
+    assert admin_user.username == "superuser"  # noqa: S101
+    assert admin_user.is_active  # noqa: S101
+    assert admin_user.is_staff  # noqa: S101
+    assert admin_user.is_superuser  # noqa: S101
+    assert admin_user.email == ""  # noqa: S101
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**chore:** The S101 noqa markers should not be necessary

That error code is about using asserts - and only applies to production code because running code with the python optimise flag on will remove asserts. It doesn't apply to tests where we need to use them all the time. This linting error should be disabled for the whole `src/tests` directory. See: `https://github.com/ackama/django-template/blob/main/template/setup.cfg`

Are you seeing this linting errors when you run `inv lint` in your projects created with this template?

The S106 is fine.